### PR TITLE
fix(Carta Giovani Nazionale): [IOACGN-59] Fixes offline-operators API request has a prefixed coordinates

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR Fixes a bug which caused the list of all merchants differs from category pre-filtered search

## How to test
Open "All the categories" merchants page and compare the result with all the other categories result

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -2563,6 +2563,8 @@ bonus:
         title: "Vuoi davvero eliminare i seguenti metodi dal Portafoglio?"
         subtitle: "Potrai aggiungerli di nuovo in qualsiasi momento, ma dovrai reinserire i dati e ripetere le procedure di autorizzazione."
   cgn:
+    generic:
+      linkError: "Non è stato possibile aprire la url al momento, riprova"
```
Comment (Undermaken) [inline locales/it/index.yml:2567]: Are we sure that the end user knows about what `url` is?
```suggestion
      linkError: "Non è stato possibile aprire il sito dell'operatore al momento, riprova"
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: locales/en/index.yml [hunk 2]
```diff
@@ -2537,6 +2537,8 @@ bonus:
         title: "Do you really want to delete the following methods from the Wallet?"
         subtitle: "You can add them again at any time, but you will have to re-enter the data and repeat the authorization procedures."
   cgn:
+    generic:
+      linkError: "The url could not be opened at the moment, please try again"
```
Comment (thisisjp) [inline locales/en/index.yml:2541]: @AliceAzzolini we need your love here

File: locales/it/index.yml (hunk 1 above)
Comment (thisisjp) [reply locales/it/index.yml:2567]: and here @AliceAzzolini

Comment (AliceAzzolini) [reply locales/it/index.yml:2567]: "In questo momento, il sito dell'operatore non è raggiungibile. Riprova più tardi."
"The merchant's site can't be reached at the moment. Please, try again later."

File: locales/en/index.yml (hunk 2 above)
Comment (AliceAzzolini) [reply locales/en/index.yml:2541]: "The site can't be reached at the moment. Please, try again later."

File: locales/it/index.yml (hunk 1 above)
Comment (CrisTofani) [reply locales/it/index.yml:2567]:
```suggestion
      linkError: "In questo momento, il sito dell'operatore non è raggiungibile. Riprova più tardi."
```

**Review by Undermaken — APPROVED** [review]

**Pull request merged**